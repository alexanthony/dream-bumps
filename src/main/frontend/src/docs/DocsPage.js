// @flow
import React from 'react';
const DocsPage = () => (
  <iframe
    src="swagger-ui/index.html"
    frameBorder="0"
    marginHeight="0"
    marginWidth="0"
    width="100%"
    height="900"
    scrolling="auto"
    target="_top"
  ></iframe>);

export default DocsPage;
