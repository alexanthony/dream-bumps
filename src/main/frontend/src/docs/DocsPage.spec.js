import React from 'react';
import { shallow } from 'enzyme';
import DocsPage from './DocsPage';
import toJson from 'enzyme-to-json';

test('Renders properly', () => {
  const component = shallow(
    <DocsPage />
  );
  expect(component).toBeDefined();
  expect(toJson(component)).toMatchSnapshot();
});
