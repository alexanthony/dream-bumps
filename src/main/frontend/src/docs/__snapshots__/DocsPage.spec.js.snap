exports[`test Renders properly 1`] = `
<iframe
  frameBorder="0"
  height="900"
  marginHeight="0"
  marginWidth="0"
  scrolling="auto"
  src="swagger-ui/index.html"
  target="_top"
  width="100%" />
`;
